src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -753,6 +800,8 @@ public static class CurrentProcessingFileInformation {
         private long startPosition;
         @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_LAST_MODIFIED_TIME)
         private long lastModifiedTime;
+        @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_LAST_ACCESSED_TIME)
+        private long lastAccessedTime = Instant.EPOCH.toEpochMilli();
```

Comment:
default should be now right? customers upgrading to this version won't have this key, so the default should be something more conservative

Reply:
I was setting the value on the ProcessingFiles but doing it here is actually simpler. Will take that suggestion.